[Botskills] Update documentation with new schema changes (#205)
southworks/botframework-solutions: tools/botskills/docs/commands/connect.md

| Option                        | Description                                                                                                                                                                         |
|-------------------------------|-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
| -l, --localManifest [path]   | Path to local Skill Manifest file                                                                                                                                                   |
| -r, --remoteManifest [url]   | URL to remote Skill Manifest                                                                                         |

> | -e, --endpointName [name]  | (OPTIONAL) Name of the endpoint to connect to your assistant (case sensitive)(defaults to using the first endpoint) (Intended V2 manifests)                                   |

Reviewer: ```suggestion
| -e, --endpointName [name]  | (OPTIONAL) Name of the endpoint to connect to your assistant which is case sensitive (defaults to using the first endpoint, used for V2 manifest)                                   |
```

| --luisFolder [path]           | (OPTIONAL) Path to the folder containing your Skills' '.lu' files (defaults to './deployment/resources/skills/en-us' inside your Virtual Assistant folder)                             |
| --dispatchFolder [path]       | (OPTIONAL) Path to the folder containing your Virtual Assistant's `.dispatch` file (defaults to `./deployment/resources/dispatch` inside your Virtual Assistant folder)          |
| --outFolder [path]            | (OPTIONAL) Path for any output file that may be generated (defaults to your Virtual Assistant's root folder)                                                                        |
| --lgOutFolder [path]          | (OPTIONAL) Path for the LuisGen output (defaults to a 'service' folder inside your Virtual Assistant's folder)                                                                      |
| --resourceGroup [name]        | (OPTIONAL) Name of your Virtual Assistant's resource group in Azure (defaults to your Virtual Assistant's bot name)                                                                 |

> | --appSettingsFile [path]      | (OPTIONAL) Path to your appsettings file (defaults to `appsettings.json` inside your Virtual Assistant's folder. Also, **the skills are stored here**)                                                                    |

Reviewer: ```suggestion
| --appSettingsFile [path]      | (OPTIONAL) Path to your appsettings file where the skills are stored (defaults to `appsettings.json` inside your Virtual Assistant's folder)                                                                    |
```

| --cognitiveModelsFile [path]  | (OPTIONAL) Path to your Cognitive Models file (defaults to `cognitivemodels.json` inside your Virtual Assistant's folder)                                                           |
| --verbose                     | (OPTIONAL) Output detailed information about the processing of the tool                                                                                                             |
| -h, --help                    | Output usage information                                                                                                                                                            |

> An example on how to use it with a local Skill manifest file:

Reviewer: ```suggestion
An example on how to use it with a local manifest file:
```

```bash

> botskills connect --localManifest "./skills/customSkill/customSkillManifest.json" --appSettingsFile "./appsettings.json" --cs

Reviewer: ```suggestion
botskills connect --localManifest "./skills/customSkill/customSkillManifest.json"  --cs
```

```

> > **Note:** The paths to both the Skill Manifest and the appsettings file, where the Skills are stored, can be relative or absolute paths equally, and should be explicitly a `.json` file.

Reviewer: ```suggestion
> **Note:** The paths to the Skill Manifest can be relative or absolute paths equally, and should be explicitly a `.json` file.
```

An example on how to use it with a remote Skill manifest:

```bash

> botskills connect --remoteManifest "http://<YOUR_SKILL_MANIFEST>.azurewebsites.net/api/skill/manifest" --appSettingsFile "./appsettings.json" --cs

Reviewer: ```suggestion
botskills connect --remoteManifest "http://<YOUR_SKILL_MANIFEST>.azurewebsites.net/api/skill/manifest" --cs
```
